azerothcore/azerothcore-wotlk
# Fix start scripts

## Changes Proposed:
-  Fix default value for `zipPath`. If no `conf/config.sh` was setup, this variable could be empty and lead to permission errors during bash installation.
   - Fixes error: `/home/user/azerothcore-wotlk/apps/installer/includes/functions.sh: line 245: /data.zip: Permission denied
`
-  Fix volume mounting. A trailing slash in the target directory of volumes lead to errors during docker installation.
   - Fixes error: `Error response from daemon: path /home/user/azerothcore-wotlk is mounted on / but it is not a shared mount.`
## Issues Addressed:

- None

## SOURCE:

- [Docker Fix](https://stackoverflow.com/questions/68273745/how-to-make-a-mount-shared-in-docker/68455665#68455665)

## Tests Performed:

- Successfully built and started on WIN10 with Git Bash (Bash and Docker installation) with no `config/config.sh`.


## How to Test the Changes:


1. Make sure you have the default configuration in `conf/dist/config.sh` and no `config/config.sh`
2. Make sure Docker does not use cached or old volumes/images
3. Follow the bash and docker installation procedures respectively
4. None of the above errors should be thrown while downloading the client-data or when building


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
